Kitodo/src/main/java/org/kitodo/services/data/HistoryService.java

```diff
@@ -229,4 +238,9 @@ public void addAllObjectsToIndex() throws InterruptedException, IOException, Cus
         indexer.setMethod(HTTPMethods.PUT);
         indexer.performMultipleRequests(findAll(), historyType);
     }
+
+    @Override
+    public HistoryDTO convertJSONObjectToDTO(JSONObject jsonObject, boolean related) throws DataException {
+        return new HistoryDTO();
```

Comment:
It is not but here I had to. In other case code would not compile - override method.